### PR TITLE
Habuild patch 1

### PR DESCRIPTION
**Pull request by habuild**
## Proposed change

I ended up on this page while searching for 'notify.persistent notification'
I was following this example and found it lacking, and some of the markdown commands either non working or hard to follow.

The previous Sample Text is in the form of json which doesn't work in dev tools and sets the service back to null when entered.
This has been changed to yaml and the related picture has been updated to a more current version.

Newline /n doesn't appear to function which requires double spacing of lines to solve.
see yaml markdown example.

Added example Markdown and HTML variable notification and related image,
This should make it clearer for new users as to see what is allowed and can be achieved.

- [X] Added example and image to the current documentation (`current` branch).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/persistent_notification.markdown [hunk 1]
```diff
@@ -100,3 +100,29 @@ Choose the **Services** tab from the **Developer Tools** sidebar item, then sele
...
+    }}/os-agent_1.4.0_linux_x86_64.deb<br></br> sudo dpkg -i os-agent_{{
+    states('sensor.home_assistant_os_agent_latest_tag')
+    }}_linux_x86_64.deb<br></br>
+
+```
+{% endraw %}
+
+This is the output.
+
+![persistant_notification_msgCapture](https://user-images.githubusercontent.com/36288425/191888076-0895e15a-2cc2-4e31-9755-c758135b108e.PNG)
```
Comment (frenck) [inline source/_integrations/persistent_notification.markdown]: Images should be included in the repository.

File: source/_integrations/persistent_notification.markdown [hunk 2]
```diff
@@ -100,3 +100,29 @@ Choose the **Services** tab from the **Developer Tools** sidebar item, then sele
 }
 ```
 This will create the notification entry shown above.
+
+### Example Markdown variable notification
```
Comment (frenck) [inline source/_integrations/persistent_notification.markdown]: What is this example adding? As in, I'm not sure what you are trying to show?
Please update the PR description to elaborate / motivate the change.

File: source/_integrations/persistent_notification.markdown [hunk 3]
```diff
@@ -100,3 +100,29 @@ Choose the **Services** tab from the **Developer Tools** sidebar item, then sele
...
+
+{% raw %}
+
+```yaml
+service: persistent_notification.create
+data:
+  notification_id: os-agent            ## <<< this can be whatever 12343232
+  title: "There is an update in Os-Agent!"
+  message: >-
```
Comment (frenck) [inline source/_integrations/persistent_notification.markdown:120]: The example title says this is a Markdown example, but it contains mainly HTML. Please remove the HTML.

File: source/_integrations/persistent_notification.markdown [hunk 4]
```diff
@@ -100,3 +100,29 @@ Choose the **Services** tab from the **Developer Tools** sidebar item, then sele
...
 This will create the notification entry shown above.
+
+### Example Markdown variable notification
+
+{% raw %}
+
+```yaml
+service: persistent_notification.create
+data:
+  notification_id: os-agent            ## <<< this can be whatever 12343232
```
Comment (frenck) [inline source/_integrations/persistent_notification.markdown]:
```suggestion
  notification_id: os-agent
```

File: source/_integrations/persistent_notification.markdown [hunk 5]
```diff
@@ -100,3 +100,29 @@ Choose the **Services** tab from the **Developer Tools** sidebar item, then sele
...
+  title: "There is an update in Os-Agent!"
```
Comment (frenck) [inline source/_integrations/persistent_notification.markdown]: Why is the OS Agent used in this example? Is there a specific reason?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/persistent_notification.markdown (hunk 3 above)
Comment (habuild) [reply source/_integrations/persistent_notification.markdown:120]: https://daringfireball.net/projects/markdown/syntax#p

That is the link from the page we are discussing.

the current example is. Go to dev tools and enter something like

```json
{
  "notification_id": "1234",
  "title": "Sample notification",
  "message": "This is a sample text"
}
```
I didn't want to remove that.

however
You can't actually do that in dev tools - services with `Persistent Notification: Create`. 
It will revert back to a blank service. 

I also found the markdown not to be working correctly. newline  `\n` doesn't seem to work



I couldn't get a lot of the markdown to work. Whereas the html would. The rest is my ignorance I should have listed it as a html example. I could do both, although not sure html is allowed(or promoted).

Try this in dev tools, I am not great at markdown, this doesn't seem to me correct though.
```
service: persistent_notification.create
data:
  notification_id: frenck
  message: >-
    Agent "{{ states('sensor.os_agent_version_github') }}" is
    >available

    ----------

    1.  Bird
    2.  McHale
    3.  Parish

    1.  Bird

    2.  McHale

    3.  Parish


    * Bird

    *  Magic
    *  Bird

    * Magic

    ---------

    <ul>
    <li>Bird</li>
    <li>Magic</li>
    </ul>

    ***    

    * * *

    - - -

    ------------------------------------


    [Latest](https://github.com/home-assistant/os-agent/releases/latest)\n


    **wget https://github.com/home-assistant/os-agent/releases/download/** "{{
    states('sensor.os_agent_version_github') }}"/os-agent_"{{
    states('sensor.os_agent_version_github') }}"_linux_aarch64.deb\n
    sudo dpkg \n -i os-agent_"{{ states('sensor.os_agent_version_github')
    }}"_linux_aarch64.deb
  title:  Hi Frenck tyvm


```

File: source/_integrations/persistent_notification.markdown (hunk 5 above)
Comment (habuild) [reply source/_integrations/persistent_notification.markdown]: No specific reason other than, that is what I was making for myself at the time, when I was following the docs.

This might help with the context
https://community.home-assistant.io/t/persisent-notifications-and-critical-notifications/464484

File: source/_integrations/persistent_notification.markdown (hunk 1 above)
Comment (habuild) [reply source/_integrations/persistent_notification.markdown]: noted. moved.

File: source/_integrations/persistent_notification.markdown (hunk 2 above)
Comment (habuild) [reply source/_integrations/persistent_notification.markdown]: updated proposed changes
